data/redirects.yml

```diff
@@ -656,6 +656,10 @@
   - /go/scout-scores/
 "/scout/explore/exceptions/":
   - "/go/scout-exceptions/"
+"/scout/explore/analysis":
```

Comment:
We're not consistent on the trailing `/` - do they matter (and should they be the same?)

Reply:
good catch!